packages/evm/hardhat.config.ts

```diff
@@ -107,6 +108,21 @@ const config: HardhatUserConfig = {
       chainId: 1101,
       url: "https://zkevm-rpc.com",
     },
+    bsc: {
+      ...sharedNetworkConfig,
+      chainId: 56,
+      url: "https://bscrpc.com",
+    },
```

Comment:
@jfschwarz  This change addresses an issue with the BSC network configuration where transactions fail due to underpriced gas. Specifically, the error `ProviderError: transaction underpriced: gas tip cap 1, minimum needed 1000000000` occurs because the gasPrice is not explicitly set.

To fix this, I added:

`gasPrice: Number(parseUnits("1", "gwei"))`,